src/eu/ldbc/semanticpublishing/refdataset/AnalyticsDataManager.java

```diff
@@ -0,0 +1,77 @@
+package eu.ldbc.semanticpublishing.refdataset;
+
+import eu.ldbc.semanticpublishing.properties.Configuration;
+
+import java.io.*;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class AnalyticsDataManager {
+
+	public static void storeInCache(Configuration configuration) {
```

Comment:
This is not cache, please rename this and the method below to something like storeInFile/loadFromFile

Reply:
ok